webapp/locales/en.json

```diff
@@ -22,14 +22,14 @@
     "follow": "Follow",
     "followers": "Followers",
     "following": "Following",
-    "shouted": "Shouted",
+    "shouted": "Recommended",
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-06-07T19:12:27Z" title="Friday, June 7th 2019, 9:12:27 pm +02:00">Jun 7, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -14,22 +14,22 @@
     "hello": "Hello"
   },
   "editor": {
-    "placeholder": "Leave your inspirational thoughts..."
+    "placeholder": "Write something inspiring..."
```


Could you justify this change? I prefer the way it is

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-06-07T22:37:24Z" title="Saturday, June 8th 2019, 12:37:24 am +02:00">Jun 8, 2019</time>_

---


Agree with @mattwr18 here